contents/node_operator.rst

```diff
@@ -412,6 +412,101 @@ If the external node is running on IP 12.34.56.78 port 8545, the command would b
+
+The balances (remember that the node maintains an independent balance with each peer it had interactions with) can be queried via RPC. In the above snippet, we started the binary with the `--ws` flag(s), which allows the node to be queried via Websockets. Here is an example of how this can be done via JSON RPC (the values here are from an example, your node may return totally different values):
+
+   .. code-block:: none
+   
+      >$ echo '{"jsonrpc":"2.0","method":"swap_balances","params":[], "id":104}' | websocat ws://localhost:8546/ -n --one-message --origin localhost | jq
+        {
+          "jsonrpc": "2.0",
+          "id": 104,
+          "result": {
+            "10e14c08a7c873adb30516807c138781da76d284dbb7f12d27d95919f9da3d24": 0,
+            "1913db38b3a9d8cf2a5110f4ab1fbd0b882f729063e456e76c874bf3bda49788": -488096051628,
+            "292fb3f158a8313c0606df126bef393c0d49f1879f35a813e07fc370a95f318c": -81349341938,
+            "2cf4f38451678a6276ab1ad7039f91c7ebe99beca2342132d80566148157702f": -744251831714,
+            "4069823dc97610784f237c5189bb0047fa44fdb58050408186e76dfc678117e4": 0,
+            "50b3e7aecb9348770b28fe846e6ccd4a65dc6a7e6fad7f32f386226fca0fbb05": -732144077442,
+            "5e2e42c0a2476cd3a5df0ba0f361a077202d0a05d8885940f5731f11bd06e314": -81349341938,
+            "817a6498af5f83c8b9bd7991523696ac82af0b19c68dd20c44dee128b04c24b4": -569445393566,
+            "8449ac2043f5eb12b859bb909ae7632fc2c64da8f5c5b0e24726bfc2b3b73683": 0,
+            "8e14175898416068388ee88ac1b51b03c64c0f32ffd02bcfc5c40dc38a8273f4": -162698683876,
+            "9e2d5a87d089d32996ccc650e29caf6827dd31349ae779d643438eb6a64bae57": 0,
+            "ac370a71c55dbae1eda0c41406f36db04967230f5a6d53a8ca00a96d560b25e9": -650794735504,
+            "b0b4283f1b00d9b12f9b57639175505d1e2db46ff553d393c6b79fbaa1eb5877": 0,
+            "c13d2f0b212b2b2b7f4414be742ed361d63725fa571c7b04c562b980242372e2": 0,
+            "c3faaf2ff61edbe4bece896467aa517989c82bcddcc9e4a0b7cc91ef991adc82": -1233143154257,
+            "d196803f49ac650598a606a3417b19c80d234e71a79c98881b9bb75359aebe6b": 0,
+            "e75b3b14877058c9f4d6884fb0be474f9d42864aa0b430fac62c46d3058ba38d": -4636912490466,
+            "ed57d9d24f898731b928466ad05aeedeb698a82ad5e7f4a86a217b37a1190bbc": -416804432270
+          }
+        }
+
+Running a SWAP-enabled node on an alternative blockchain
+=========================================================
+If you want to run Swarm with a different blockchain platform (for example, RIF Storage runs a Swarm network on their own RSK network, check [https://www.rifos.org/blog/rif-storage-testnet-launch], or you may want to run experiments and tests within your own development environment, say using `ganache`), you need to deploy the factory contract and provide its address to the options via `-swap-chequebook-factory` 
+
+
+//TODO: Add details of how to deploy honey ERC20 contract, how to fund addresses etc.
```

Comment:
Hmm. I'm not sure if such details is relevant for the "node operator"

Reply:
OK maybe you are right. @mortelli @ralph-pichler please confirm that you agree.

Is this documented somewhere though? Maybe a hackmd or something

Reply:
Not really relevant for node operators but it still might be useful to have a section for people that need to deploy their own swarm network. The easiest way to do this is to use the generic deployment script but it is not exactly well documented.

Reply:
agree with @ralph-pichler